wiki/fr/fr-Jamulus-Cloud.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: wiki
+title: "Jamulus in the Cloud"
```

Comment:
No need to translate this file (will be removed soon).

Reply:
OK.